apps/pie-microsite/src/assets/styles/base/_layout.scss

```diff
@@ -34,6 +35,18 @@
 .c-main {
     grid-area: main;
     position: relative;
+
+}
+
+.c-content-container {
```

Comment:
(Don't hate me for the post-merge comment – just catching up on PRs as it's been a busy week 😆 )

As there isn't a `c-content` class, this should be `c-contentContainer`, otherwise, it breaks our naming scheme. If you could update this in a future PR, that'd be 🚀